frontend/src/txns/execute.ts

```diff
@@ -1,10 +1,94 @@
 import { OfflineAminoSigner, OfflineDirectSigner } from '@keplr-wallet/types';
 
+import {
+  SigningStargateClient,
+  defaultRegistryTypes,
+  StdFee,
+  DeliverTxResponse,
+} from '@cosmjs/stargate';
+import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx';
+import { GeneratedType, Registry } from '@cosmjs/proto-signing';
+import { MsgClaim } from './passage/msg_claim';
```

Comment:
We are not going to have passage specific feature in new dashboard. Let's remove all passage related code.